build.sbt

```diff
@@ -12,7 +12,7 @@ lazy val jsonrpc = project.in(file("jsonrpc"))
   .settings(
     name := "scalabrad-web-jsonrpc",
 
-    resolvers += "Typesafe repository" at "http://repo.typesafe.com/typesafe/releases/",
+    resolvers += Resolver.jcenterRepo,
```

Comment:
Let's add this in `commonSettings` above, which currently has `"bintray" at "http://jcenter.bintray.com"`; can just replace that with `Resolver.jcenterRepo`. The `commonSettings` are included here in `jsonrpc` and also in `server` below.